src/configure-tests/feature-tests/genhd_fl_no_part.c

```diff
@@ -0,0 +1,14 @@
+// SPDX-License-Identifier: GPL-2.0-only
+
+/*
+ * Copyright (C) 2015 Datto Inc.
```

Comment:
You can change the date from 2015 to 2023

Reply:
done :)